lib/hap/services/air_quality_sensor.ex

```diff
@@ -0,0 +1,32 @@
+defmodule HAP.Services.AirQualitySensor do
+  @moduledoc """
+  Struct representing an instance of the `public.hap.service.sensor.air-quality` service
+  """
+
+  defstruct air_quality: nil, name: nil, active: nil, fault: nil, tampered: nil, low_battery: nil
+
+  defimpl HAP.ServiceSource do
+    def compile(value) do
+      HAP.Service.ensure_required!(__MODULE__, "air_quality", value.air_quality)
+
+      %HAP.Service{
+        type: "8A",
```

Comment:
Per p.134 of HAP R2 this is 8D, no?

Reply:
Thanks, this also explained why the Home App didn't display the Sensor correctly 🙃